enet.pyx

```diff
@@ -252,7 +252,7 @@ cdef class Address:
     def __init__(self, host, port):
         if host is not None:
             # Convert the hostname to a byte string if needed
-            self.host = host if isinstance(host, bytes) else bytes(host, "ascii")
+            self.host = host if isinstance(host, bytes) else host.encode('cp437')
```

Comment:
I don't think this is good.
'cp437' encoding is very specific to our case.

I'm not sure the previous encoding of "ascii" is any better though.

What I would do here is just do `host.encode()`, which will work under both py2 and py3